lib/rucio/daemons/replicarecoverer/suspicious_replica_recoverer.py

```diff
@@ -68,7 +69,7 @@
 GRACEFUL_STOP = threading.Event()
 
 
-def declare_suspicious_replicas_bad(once=False, younger_than=3, nattempts=10, vos=None, limit_suspicious_files_on_rse=5, sleep_time=300):
+def declare_suspicious_replicas_bad(once=False, younger_than=3, nattempts=10, vos=None, limit_suspicious_files_on_rse=5, sleep_time=3600, passive_mode=True):
```

Comment:
What should be the default case? In the argument parser it's `False`, here `True`. This should be consistent.

Reply:
The default case should be passive_mode=True, so that no accidents happen. Unfortunately, we currently only have the option to test the suspicious replicas recoverer on production servers, so it is very important that no actions are taken by mistake.

Reply:
Alright, I've changed it to be "--disable-passive-mode", as I think this is a bit clearer.